### DISCUSSION TITLE
Checkbutton style change

### DISCUSSION THREAD

**Discussion by youstanzr** [israel-dryer/ttkbootstrap#289] [Q&A]
Hi,
Is it possible to change the background color a checkbutton or make it the background transparent?

Also is it possible to increase the size of it?
currently changing the size only changes the background but the checkbox remains same

Current:
![cb](https://user-images.githubusercontent.com/13307813/168790514-6fffbf6a-e7b9-4eb7-8113-7a804646c21d.png)

Goal:
![cb_goal](https://user-images.githubusercontent.com/13307813/168790717-b49cdfe4-3c5f-4953-b85c-131081062036.png)

Thanks

**Comment by israel-dryer** [comment]
Yes you can do this by modifying the style that you want to use on the checkbutton. Under the hood, a bootstyle is just a pre-defined ttk style; so you can subclass it to make your own.

```python
import ttkbootstrap as ttk
from ttkbootstrap.constants import *

# create a new window
app = ttk.Window(themename='flatly', size=(500, 400))

frame = ttk.Frame(app, bootstyle='primary', padding=10)
frame.pack(fill=X, padx=10, pady=10, side=TOP)

lbl = ttk.Label(frame, text='Search', bootstyle='primary-inverse')
lbl.pack(side=LEFT)

# create a custom style by subclassing an existing ttkbootstrap style
app.style.configure('my.secondary.TCheckbutton', background=app.style.colors.primary)

# create a checkbutton and apply the new style with the `style` argument
cb = ttk.Checkbutton(frame, style='my.secondary.TCheckbutton')
cb.pack(side=LEFT, padx=10)

app.mainloop()
```
<img alt="python_NGw7zk7o3F" src="https://user-images.githubusercontent.com/42584742/169730580-f5827edb-08a3-4497-b327-db4dfba0981d.png">

**Reply by youstanzr** [reply]
That works but it causes loss of bootstyle change to be success for example

**Reply by youstanzr** [reply]
Is there a way to apply the style and bootstyle of success or maybe apply the bootstyle into the style so when styling the widget, it automatically applies the proper bootstyle?

**Reply by israel-dryer** [reply]
If you are using a version of the success bootstyle, you can just subclass the success bootstyle.  "success.TCheckbutton"
